tools/analysis/analysis_runner.py

```diff
@@ -0,0 +1,77 @@
+# Analysis runner.
+import argparse
+import os
+import subprocess
+
+# TODO: Support customizing these.
+_TOPIC = 'projects/ossf-malware-analysis/topics/workers'
+_PACKAGES_BUCKET = 'ossf-malware-analysis-packages'
+
+
+def _upload_file(local_path):
+  # TODO: figure out a better way to key these.
```

Comment:
You could just split the filename and extension and suffix the filename with some random value (or timestamp or uuid).

e.g:
foo-1.6.1.tgz becomes foo-1.6.1-m21h3kj12.tgz

Reply:
I was thinking this needs more thought, because we might want this bucket to become a more general store for all artifacts our infrastructure sees. e.g. we might want to store all package versions we see from package-feeds in this bucket. Maybe we want something like `gs://bucket/package/version/hash.ext` ? 

For now just filename seems good enough for our immediate experimental use case. We can easily change this later.

Reply:
Yeah, I like the idea of having a general artifact store for everything. Naming is tricky and we may also want `/bucket/package/version/platform/hash.ext`. For now doing something easy for experimenting sounds good.